src/pal/src/cruntime/printfcpp.cpp

```diff
@@ -1471,7 +1456,7 @@ int CoreVfwprintf(CPalThread *pthrCurrent, PAL_FILE *stream, const wchar_16 *for
                     va_list apcopy;
 
                     va_copy(apcopy, ap);
-                    TempInt = NativeVsnprintf(pthrCurrent, TempSprintfStr, TEMP_COUNT, TempBuff, apcopy);
+                    TempInt = vsnprintf(TempSprintfStr, TEMP_COUNT, TempBuff, apcopy);
```

Comment:
It would make sense to use the standard snprintf and drop support for VS2013 and older. @jkotas Do you agree?